src/reticulum.js

```diff
@@ -146,6 +140,7 @@ class ReticulumChannel extends EventEmitter {
 class ReticulumClient {
 
   constructor(hostname) {
+    this.sessionIds = new Set(); // all of the session IDs that belong to ourselves
```

Comment:
This seems to be unused.

Reply:
Nice catch!

Reply:
In my defense I also added things that were used.